src/outro.js

```diff
@@ -1,2 +1,13 @@
+for (var key in MathQuill) (function(key, val) {
+  if (preInterVerMathQuill[key]) return; // already set .noConflict
+  if (typeof val === 'function') {
+    preInterVerMathQuill[key] = function() {
+      insistOnInterVer();
```

Comment:
> I'm a little confused about how this will work out in practice. Does this mean that people who don't specify an interfaceVersion will see a console warning for each API call they make?

Yes.

> Is it considered valid to not declare an interface version until you are ready to opt in to version 1? Or is version 1 effectively the same as the pre InterVer behavior, and we're expecting that everyone that gets on "MathQuill 2.0" (if that's what we're calling the dev branch once it's released) should definitely immediately specify an InterVer version.

Declaring interface version 1 changes no API behavior except to stop the warnings.

The idea is basically that the easiest thing to do with an API should be the thing that the API's designers want the vast majority of people to do. Things intended for a minority shouldn't be the path of least resistance. That's why declaring an interface version has to be required, we can't just have omitting an interface version be equivalent to declaring the pre-InterVer interface version: the pre-InterVer interface version would be the path of least resistance.

All you have to do to stop the warnings is add that one `MathQuill.interfaceVersion(1);` call. Given that, do you feel strongly about reducing the warnings?